docs/docsite/rst/getting_started/index.rst

```diff
@@ -1,99 +1,127 @@
-.. _getting_started_index:
+Getting Started with Ansible in a Self-Contained Project Structure
+==================================================================
 
-############################
-Getting started with Ansible
-############################
+Introduction to Ansible
+-----------------------
 
-Ansible automates the management of remote systems and controls their desired state.
-A basic Ansible environment has three main components:
+Ansible is an open-source automation platform that allows IT professionals to automate various areas of their domain, for example:
 
```

Comment:
I'm not sure why we are repeating an intro to ansible here? Can't we just point to the existing intro page?

Reply:
Should be fixed in a new PR: https://github.com/ansible/ansible-documentation/pull/14